README.md

```diff
@@ -139,14 +139,25 @@ Type: `Hypercore`
 Add a hypercore to the indexer. Must have the same value encoding as other
 hypercores already in the indexer.
 
+Rejects if called after the indexer is closed.
+
+### indexer.idle()
```

Comment:
This method was previously undocumented. Because it has a behavior change, I documented it here.